### DISCUSSION TITLE
User management in a new prj

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#366] [Q&A (Help)] [answered]
When creating a new project with BackPack (v5.4.14) and DevTools a User model is created which saves the password in plain text, am I wrong? What is the best solution instead?
Another thing, the controller UserCrudController is automatically created which allows the management of users but if I enter in edit and save I am forced to re-enter the password again. Wouldn't it be better to manage the password change separately or, in case of an empty password field, save the previous value?

**Comment by blondie63** [comment]
I think i make a mistake, by default a backpack project use default bcrypt, i think..
But there's no info on help..

**Comment by tabacitu** [comment, marked as answer]
New projects start off with a widget on the Dashboard that explains how to create a Users CRUD. Please follow the instructions there, they cover everything you wanted and more. You can also see the instructions in HTML here: https://github.com/Laravel-Backpack/CRUD/blob/main/src/resources/views/base/inc/getting_started.blade.php
